selfdrive/locationd/test/test_locationd.py

```diff
@@ -8,6 +8,7 @@
 
 from cereal import log
 import cereal.messaging as messaging
+from cereal.services import services
```

Comment:
services isn't supposed to be used directly, it's just for building service_list

Reply:
I want the various message frequencies though. Where should I read that from?